graphql/schema.graphql

```diff
@@ -269,6 +272,11 @@ type Query {
   """
   order(id: String!): Order @cacheControl(scope: PRIVATE)
 
+  """
+  Returns the specified site configurations
+  """
+  siteConfigs: SiteConfigs @cacheControl(scope: PRIVATE)
```

Comment:
Maybe storeConfigs is a better option or even CheckoutConfigs...

Reply:
Don't think you need the cacheControl private, hence you are using the authToken